aloscene/depth.py

```diff
@@ -22,18 +24,55 @@ class Depth(aloscene.tensors.SpatialAugmentedTensor):
     """
 
     @staticmethod
-    def __new__(cls, x, occlusion: Mask = None, *args, names=("C", "H", "W"), **kwargs):
+    def __new__(
+            cls, 
+            x, 
+            occlusion: Mask = None, 
+            is_inverse=False, 
+            scale=None, 
+            shift=None, 
+            *args, 
+            names=("C", "H", "W"), 
+            **kwargs):
+        if is_inverse and not (shift and scale):
+            raise AttributeError('inversed depth requires shift and scale arguments')
         if isinstance(x, str):
             x = load_depth(x)
             names = ("C", "H", "W")
         tensor = super().__new__(cls, x, *args, names=names, **kwargs)
         tensor.add_child("occlusion", occlusion, align_dim=["B", "T"], mergeable=True)
-
+        tensor.add_property('_scale', scale)
+        tensor.add_property('_shift', shift)
+        tensor.add_property('_is_inverse', is_inverse)
         return tensor
 
     def __init__(self, x, *args, **kwargs):
         super().__init__(x)
 
+    def encode_inverse(self, scale: float, shift: float):
+        """Shift and scales the depth values"""
```

Comment:
Here is an example of expected doc string  
```
"""Attach an occlusion mask to the depth tensor.
  Parameters
  ----------
  occlusion: Mask
      Occlusion mask to attach to the Frame
  name: str
      If none, the occlusion mask will be attached without name (if possible). Otherwise if no other unnamed
      occlusion mask are attached to the frame, the mask will be added to the set of mask.
  """
```